modules/blog/author-detail/author-detail.tsx

```diff
@@ -19,11 +19,18 @@ const AuthorDetail = ({ authorInfo }: Props) => {
           height={authorInfo.picture.height}
           src={authorInfo.picture.url}
           width={authorInfo.picture.width}
+          style={{
+            maxWidth: '100%',
+            height: 'auto',
```

Comment:
As Vlad said, it probably makes sense to but this style object in a `const` and reuse it.

Reply:
You could probably put it in [this file](https://github.com/kedro-org/kedro-website/blob/main/utils/blog/index.ts).

Reply:
@tynandebold @vladimir-mck moved thanks.